translations/src/config.js

```diff
@@ -25,7 +25,7 @@ class Config {
       additionalFont: 'lateef'
     },
     en: { rtl: false },
-    fa: {
+    pes: {
```

Comment:
Not yet in our wiki afaict: https://wiki.tuerantuer.org/integreat-languages
Could you check and add it there?

Reply:
afais it is in our wiki. Check out the row with a `fa` as slug